lib/fastlane/plugin/wpmreleasetoolkit/helper/ios/ios_version_helper.rb

```diff
@@ -1,3 +1,5 @@
+require 'xcodeproj'
```

Comment:
We need to make sure this gem is listed in the toolkit's gemspec.

Even if in practice, because the release-toolkit plugin will only make sense when used with fastlane, and fastlane already has xcodeproj as a transitive dependency, imho it still makes sense to make it explicit that we our tookit gem on the that xcodeproj gem.

Reply:
Agreed 👍

Reply:
Added here https://github.com/iangmaia/release-toolkit/pull/1/commits/09ade61cdf715da1f381e56d93c2ecc4f2e092c6